CONTRIBUTING.md

```diff
@@ -303,8 +303,6 @@ To merge changes into `main`, a feature branch needs to satisfy the following co
 * Passing code check, e.g. no trailing white spaces, proper Doxygen style, ...
 * No build errors and warnings
 * All tests are passing.
-* Approval of the changes by at least one respective code owner (
-  cf. `<path/to/four-c-source-code>/.gitlab/CODEOWNERS.md`)
```

Comment:
It is not super relevant since we have #171, but you could mention the two required approvals by developers.